srcDLL/DllMain.cpp

```diff
@@ -42,9 +42,9 @@ BOOL WINAPI DllMain(HINSTANCE hInstance, DWORD dwReason, LPVOID /*reserved*/)
 	// This will be called once the program is unpacked and running
 	if (dwReason == DLL_PROCESS_ATTACH) {
 		// Setup logging
-		SetLoggerError(&loggerDistributor);
-		SetLoggerMessage(&loggerFile);
-		SetLoggerDebug(&loggerDebug);
+		SetLogDestinationError(&logDistributor);
+		SetLogDestinationMessage(&logFile);
+		SetLogDestinationDebug(&logDestinationDebug);
```

Comment:
I'm not a huge fan of this rename, though maybe I just need to warm up to it. It's more precise to include "Destination", though it feels kind of long and wordy.

Reply:
I agree it is wordy but also more precise. I'm okay with either leaving or pulling the word Destination.

Reply:
I kind of want to pull it here. I find when its there, it changes how I break up and parse the words, and they make less sense. It's like I'm putting a slight pause between "Log" and "Destination". Mentally grouping "Destination" with "Error", "Message", or "Debug" just doesn't flow well, and makes the name sound a bit confusing. If someone else had no experience with the API, they might be tempted to parse the names in the same way.